src/Cms/App.php

```diff
@@ -543,6 +543,15 @@ public function csrf(string|null $check = null): string|bool
 		return false;
 	}
 
+	/**
+	 * Checks if CORS support is enabled
+	 * @since 5.2.0
+	 */
+	public function isCorsEnabled(): bool
```

Comment:
`Thought`: The `App` class already covers a lot of features and therefore has quite many public methods. This makes the class hard to maintain and to keep an overview of all functionality.

`Suggestion`: With the new `Cors` class, we could move this method to `Cors::isEnabled()`.

Reply:
I think this is something that we should agree upon first as a team because @distantnative and I already let @johannschopplich refactor this part before with a new method name :) I would not mind another method in the App class, but the `Cors::isEnabled()` pattern would also replicate what we do with `UUIDs` so I definitely see the benefits of that as well.

Reply:
We decided to keep it like this as it's overall simpler due to the access to the `App` object.